### PR TITLE
Remove deprecated API

### PR DESCRIPTION
**Pull request by emartynov**
## Task
Remove deprecated API part of #620

### CODE REVIEW

File: reaktive/src/commonTest/kotlin/com/badoo/reaktive/completable/CompletableToCompletableTests.kt [hunk 1]
```diff
@@ -23,9 +23,6 @@ class CompletableToCompletableTestsImpl(
     transform: Completable.() -> Completable
 ) : CompletableToCompletableTests, SourceTests by SourceTestsImpl(TestCompletable(), { transform().test() }) {
 
-    @Deprecated("Just to fix complilation issues")
-    constructor() : this({ throw UnsupportedOperationException() })
```
Comment (arkivanov) [inline reaktive/src/commonTest/kotlin/com/badoo/reaktive/completable/CompletableToCompletableTests.kt:27]: Hmm, it's not part of the public API. But there were compilation errors a while ago. Does it compile across all targets?

**Comment by arkivanov** [comment]
Also could you please run apiDump Gradle task and commit the updated api files? Otherwise build will fail.

**Comment by arkivanov** [comment]
Also could you please run apiDump Gradle task and commit the updated api files? Otherwise build will fail.

File: reaktive/src/commonTest/kotlin/com/badoo/reaktive/completable/CompletableToCompletableTests.kt (hunk 1 above)
Comment (emartynov) [reply reaktive/src/commonTest/kotlin/com/badoo/reaktive/completable/CompletableToCompletableTests.kt:27]: Not sure, let me check. The JVM was compiling fine.

**Comment by emartynov** [comment]
@arkivanov dump is updated and CI looks green. Not sure what compilation error it was before.

**Review by arkivanov — APPROVED** [review]

**Comment by arkivanov** [comment]
Thanks for you contribution!

**Pull request merged**